src/it/resources/org/checkstyle/suppressionxpathfilter/methodlength/SuppressionXpathRegressionMethodLengthSingleToken.java

```diff
@@ -0,0 +1,10 @@
+package org.checkstyle.suppressionxpathfilter.methodlength;
+
+public class SuppressionXpathRegressionMethodLengthSingleToken {
+
+    public static void methodOne() { }
+
+    public void methodTwo() { // warn
```

Comment:
SuppressionXpathRegressionMethodLengthNoEmptyLines  already test violation o suppression on method of class, please make this method to have anonymous class and make violation on method of anonymous class.

Reply:
done